File: marios/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{

    int height;

    do
    {
        height = get_int("Height of pyramid: ");

    }
    while (height < 0 || height > 23);

    for(int i = 0; i < height; i++)
    {

        // Print the empty spaces to push pyramid to right
        for (int k = 1; k <= height - 1 - i; k++)
        {
            printf(" ");
        }

        // Print hashtags for left pyramid
        for (int j = 1; j < i + 1 + 1; j++)
        {

            printf("#");
        }

        // Print gap between pyramids
        printf("  ");

        // Print hashtags for right pyramid
        for (int j = 1; j < i + 1 + 1; j++)
        {

            printf("#");
        }

        // Start a new line
        printf("\n");

    }

}

File: pset6/marios/less/mario.py
from cs50 import get_int

def main():

    while True:
        height = get_int("Height between 1 and 8: ")

        if (height >= 0 and height <= 8):
            break;

   # halfPyramid(height)


#def halfPyramid(height):

    for i in range(0, height):

        for j in range(1, height - i):
            print(" ", end = "")
            print("#" * i, end = "")

       # for k in range(i + 1):
       #     print("#", end = "")

        print()


if __name__ == "__main__":
    main()